pallets/phala/src/compute/computation.rs

```diff
@@ -280,71 +281,71 @@ pub mod pallet {
 		///
 		/// Indicates a change in [`CoolDownPeriod`].
 		CoolDownExpirationChanged { period: u64 },
-		/// A miner starts mining.
+		/// A worker starts computing.
 		///
 		/// Affected states:
-		/// - the miner info at [`Miners`] is updated with `MiningIdle` state
-		/// - [`NextSessionId`] for the miner is incremented
-		/// - [`Stakes`] for the miner is updated
-		/// - [`OnlineMiners`] is incremented
-		MinerStarted {
-			miner: T::AccountId,
+		/// - the worker info at [`Workers`] is updated with `WorkerIdle` state
+		/// - [`NextSessionId`] for the worker is incremented
+		/// - [`Stakes`] for the worker is updated
+		/// - [`OnlineWorkers`] is incremented
+		WorkerStarted {
+			session: T::AccountId,
 			init_v: u128,
 			init_p: u32,
 		},
-		/// Miner stops mining.
+		/// Worker stops computing.
 		///
 		/// Affected states:
-		/// - the miner info at [`Miners`] is updated with `MiningCoolingDown` state
-		/// - [`OnlineMiners`] is decremented
-		MinerStopped { miner: T::AccountId },
-		/// Miner is reclaimed, with its slash settled.
-		MinerReclaimed {
-			miner: T::AccountId,
+		/// - the worker info at [`Workers`] is updated with `WorkerCoolingDown` state
+		/// - [`OnlineWorkers`] is decremented
+		WorkerStopped { session: T::AccountId },
+		/// Worker is reclaimed, with its slash settled.
+		WorkerReclaimed {
+			session: T::AccountId,
 			original_stake: BalanceOf<T>,
 			slashed: BalanceOf<T>,
 		},
-		/// Miner & worker are bound.
+		/// Worker & worker are bound.
 		///
 		/// Affected states:
-		/// - [`MinerBindings`] for the miner account is pointed to the worker
-		/// - [`WorkerBindings`] for the worker is pointed to the miner account
-		/// - the miner info at [`Miners`] is updated with `Ready` state
-		MinerBound {
-			miner: T::AccountId,
+		/// - [`SessionBindings`] for the worker account is pointed to the worker
```

Comment:
“the miner account” is the virtual account generated for the mining session. So we'd better say "for the session account"